### PR TITLE
Add suspension banner

### PR DESCRIPTION
**Pull request by iskandersamerkhanov**
👓 @pwildani @schwers @tomsoir 
JIRA Ticket: https://reddit.atlassian.net/browse/AE-983?filter=-1

### CODE REVIEW

**Comment by iskandersamerkhanov** [comment]
screenshot from 2017-07-17 15-12-11

**Comment by iskandersamerkhanov** [comment]
screenshot from 2017-07-17 15-20-16

File: src/app/components/SuspensionBanner/styles.less [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    color: white;
+    background-color: #ea0027;
+  }
+
+  &__text {
+    line-height: 20px;
+    margin-left: 60px;
+    padding: 10px;
+    color: white;
+    background-color: #eb758b;
```
Comment (schwers) [inline src/app/components/SuspensionBanner/styles.less]: 🔕  Might be able to use one of the nsfw salmon or error red's here. Non-blocker

**Comment by kjoconnor** [comment]
`susspension` -> `suspension`

**Comment by iskandersamerkhanov** [comment]
Fixed typo in commit name
Changed colors to '@red' and '@nsfw-salmon'

@schwers Could you please deploy this commit, cause i have not write access?

screen shot 2017-08-07 at 11 56 54 am

**Comment by schwers** [comment]
merged via https://github.com/reddit/reddit-mobile/commit/076b426e9f29210c7d72fcf695cdc03b9e9a915c

**Pull request closed without merging**